lib/src/Base/Func/PiecewiseHermiteEvaluation.cxx

```diff
@@ -99,9 +104,31 @@ Point PiecewiseHermiteEvaluation::operator () (const Point & inP) const
   if (inP.getDimension() != 1) throw InvalidArgumentException(HERE) << "Error: expected an input point of dimension 1, got dimension=" << inP.getDimension();
   const Scalar x = inP[0];
   UnsignedInteger iLeft = 0;
-  if (x <= locations_[iLeft]) return values_[iLeft];
+  if (x <= locations_[iLeft]) 
+  {
+    if (enableExtrapolation_)
+    {
+      return values_[iLeft];
+    }
+    else
+    {
+       throw InvalidArgumentException(HERE) << "Error : input point is less than the lower bound of the design of experiments=" << locations_[iLeft];
```

Comment:
What is the design of experiments in this context? Mention the locations instead of the design of experiments, which is very context specific. For example, this class is used internally to speed up some computation and the term discretization would be more appropriate.